core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.ts

```diff
@@ -98,7 +99,7 @@ export class DotEditContentFormComponent implements OnInit {
      *
      * @memberof DotEditContentFormComponent
      */
-    changeValue = output<Record<string, string>>();
+    changeValue = output<Record<string, string | string[] | Date>>();
```

Comment:
Is it possible to use the FormValues type here?

Reply:
Changed.